### ISSUE TITLE
Minimum range?

### ISSUE THREAD

**Issue by ptdev** [nathanreyes/v-calendar#256]
Hi there, 

Apologies if this is already possible with the many configuration and event options available on this component but I'm still kind of getting started with vue so I couldn't really figure it out.

I was wondering if there's any way to implement some sort of "minimum range" when using the calendar in picker mode?

For example, if the minimum range is 5, then once a user selects the first day on the picker, then the following 5 days would automatically get disabled.

This could be useful when using this calendar for bookings and/or reservations and there's a minimum stay or minimum days for something.

Is there anyway to do this with this component? Some sort of "min-range" prop would be ideal or maybe the same behavior can be achieved using the drag event and some vue trickery? Any pointers would be greatly appreciated.

Thanks.

**Comment by nathanreyes** [comment]
The way I would implement this is to watch the value that you are binding to. When it changes, modify the disabled dates that get passed into the component as well.
```html
<v-date-picker v-model="value" mode="range" :disabled-dates="disabledDates" />
```
```js
export default {
  data() {
    return {
      value: null,
      disabledDates: [],
    };
  },
  watch: {
    value(val) {
      // ...update the disabled dates here
      const fromDate = new Date(val);
      const toDate = // add 5 days to the val date
      disabledDates = [{ start: fromDate, end: toDate }]
    }
  }
}
```
I would be very interested to see if this works 😄

**Comment by piep14** [comment]
Hello,

I am looking to also do a min-range but I have not found in the documentation.

Is it possible ?

Thank you

**Comment by ptdev** [comment]
Hi, we didn't really figure out how to do it right inside the calendar unfortunately although I'm pretty sure it's probably doable with the drag event. We had to move on with the project so I didn't really spend a lot of time on this. I still believe min/max range props are a good thing to have on a date range picker component so that was the point of my initial suggestion.

So, we've opted with running some validations on the input event that compares the end-start date difference against the minimum days/nights required and according to the result then show/hide a validation message and enable/disable the submit button accordingly. It works well, although for UX it would be preferable to have it immediately displayed on the calendar once one selects the start date.

I'm closing this since it was just a feature request and there's no point in keeping it as an open issue.
